go.mod

```diff
@@ -33,6 +33,7 @@ require (
 	github.com/go-openapi/validate v0.20.1 // indirect
 	github.com/go-test/deep v1.0.4 // indirect
 	github.com/golang-jwt/jwt v3.2.1+incompatible
+	github.com/google/go-cmp v0.5.6
```

Comment:
this is also being used indirectly by `gotest.tools`, I needed it to display the diff in case comparison in test fails